pyproject.toml

```diff
@@ -14,6 +14,18 @@ packages = [{ include="cognite", from="." }]
 [tool.ruff.pyupgrade]
 keep-runtime-typing = true
 
+[tool.pydoclint]
```

Comment:
Can we configure it using cmd line args in the pre-commit file instead? We prefer putting it there, only in here if that doesn't work.

Reply:
Absolutely 👌